adcm_aio_client/core/objects/cm.py

```diff
@@ -340,6 +341,14 @@ class HostsAccessor(PaginatedAccessor[Host, None]):
     class_type = Host
 
 
+class HostsNode(HostsAccessor):
+    async def add(self: Self, provider: HostProvider, name: str, cluster: Cluster | None = None) -> None:
```

Comment:
* I think, this method should be `create()`
* Add `HostsNode` to `Client`
* add `description` field to method